### DISCUSSION TITLE
Installing typewritten with zinit

### DISCUSSION THREAD

**Discussion by chertik77** [reobin/typewritten#158] [Q&A]
How to properly install typewritten with zinit? Didn't find solution in the docs

```shell
zinit ice compile'(pure|async).zsh' pick'async.zsh' src'pure.zsh'
zinit light reobin/typewritten
```

here it says no such file or directory, any ideas? nice theme, btw

**Comment by reobin** [comment]
👋 @chertik77 !

The corrected config would be:

```sh
zinit ice compile'(typewritten|async).zsh' pick'async.zsh' src'typewritten.zsh'
zinit light reobin/typewritten
```

**Reply by chertik77** [reply]
> 👋 @chertik77 !
> 
> The corrected config would be:
> 
> ```shell
> zinit ice compile'(typewritten|async).zsh' pick'async.zsh' src'typewritten.zsh'
> zinit light reobin/typewritten
> ```

I've done a sort of test for my zsh first load with your theme, different configurations and without.

Without your theme first load is: 0.07
With this configuration: `zinit light reobin/typewritten` first load is: 0.14

With this configuration: `zinit ice compile'(typewritten|async).zsh' pick'async.zsh' src'typewritten.zsh'
zinit light reobin/typewritten` first load is: 0.14

So based on these results nothing is changed, what can cause in your theme to slow down startup of my terminal?

I use this function to see results: `for i in $(seq 1 10); do /usr/bin/time $SHELL -i -c exit; done`
